st2actions/st2actions/runners/python_action_wrapper.py

```diff
@@ -114,14 +114,27 @@ def __init__(self, pack, file_path, parameters=None, user=None, parent_args=None
     def run(self):
         action = self._get_action_instance()
         output = action.run(**self._parameters)
+        action_status = None
+
+        if type(output) is tuple and len(output) is 2:
```

Comment:
Safer to use `==` when comparing strings and numbers (`is` should only be used for built in types such as True, False, None).

Well in theory `is` should also work fine for small integers since they tend to always contain the same reference, but it's still a bit dangerous and implementation specific.